<launch>
    <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find global_planner)/logger.config"/>

    <arg name="tf_prefix" default=""/>
    <arg name="video_device" default="/dev/video1" />

    <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" >
        <param name="video_device" value="/dev/video0" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="$(arg tf_prefix)/usb_camera_link" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />

        <!-- Remap from usb_cam topics to camera topics -->
        <remap from="usb_cam/image_raw" to="usb_camera/image_raw" />
    </node>
</launch>
